MAINTAINERS.md

```diff
@@ -1,17 +1,166 @@
+| Can request rework of RFCs<br> in any part of the project                                                                                   |            ✗            |            ✗           |            ✓            |
+| Can manage release process of the project                                                                                                   |            ✗            |            ✗           |            ✓            |
+| Can represent the project in public as a Maintainer                                                                                         |            ✗            |            ✗           |            ✓            |
+
+These roles are merit based. Refer to the corresponding section for specific
+requirements and the nomination process.
+
+## <a name="contributor"></a>Contributor
+
+This role is open to anyone that would like to become actively involved in the oneDPL project. A Contributor can invest
+their time and resources in several different ways to improve oneDPL
+* Answering questions from community members in Discussions on Issues
+* Providing feedback on RFC pull requests and Discussions
+* Reviewing and/or testing pull requests
+* Contribute code, including bug fixes, tests, new examples of oneDPL use, and new feature implementations
+* Contribute design proposals as RFCs
+
+Responsibilities:
+* Follow the [Code of Conduct](CODE_OF_CONDUCT.md)
+* Submit issues, feature requests, and code in accordance with the [Contribution Guidelines](CONTRIBUTING.md)
+
+Privileges:
+* Code contributions recognized in the oneDPL [credits](CREDITS.txt)
+* Eligible to become a Code Owner
+* Read permissions granted to the repository
+* Can suggest PRs and issues to be included in Milestones during planning
+
+## <a name="codeowner"></a>Code Owner
+
+A Code Owner is an established contributor that is recognized by active code owners and maintaininers as capable of
+taking on additional activities and the responsibilities and privileges that come with them. A Code Owner is
+responsible for a specific oneDPL component or functional area of the project. Code Owners are collectively responsible,
+with other Code Owners, for developing and maintaining their component or functional areas, including reviewing all
+changes to the corresponding areas of responsibility and indicating whether those changes are ready to be merged. Code
+Owners have a track record of code contribution and reviews in the project.
+
+Requirements:
+* Track record of accepted code contributions to a specific project component.
+* Track record of contributions to the code review process.
+* Demonstrated in-depth knowledge of the architecture of a specific project
+  component.
+* Commits to being responsible for that specific area.
+* Can propose new RFC or participate in review of existing RFCs
```

Comment:
The bullet about RFCs seem to be here by accident.

Reply:
You're correct. I removed it.